### PR TITLE
[KYUUBI #550] Test jdk11 in GA

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

Test jdk11 in GA

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiSQLExceptionSuite.scala [hunk 1]
```diff
@@ -35,7 +35,6 @@ class KyuubiSQLExceptionSuite extends KyuubiFunSuite {
 
     val e3 = KyuubiSQLException(e2.toTStatus)
     assert(e3.getMessage === e2.getMessage)
-    assert(e3.getStackTrace === e2.getStackTrace)
```
Comment (turboFei) [inline kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiSQLExceptionSuite.scala:38]: image
In jdk11, the stack trace is not absolute same.

**Comment by turboFei** [comment]
ok,
